### PR TITLE
Updates based on persistent keystore

### PR DESCRIPTION
**Pull request by aphelionz**
This PR:
- Implements the `orbit-db-storage-adapter` at the test level for creating keystores
- Removes the creation of a default keystore, but it does use the passed-in keystore as the `signingKeystore` automatically.

Requires https://github.com/orbitdb/orbit-db-keystore/pull/34

### CODE REVIEW

**Comment by RichardLitt** [comment]
Should we add a WIP label to PRs, in general, actually...

File: src/identities.js [hunk 1]
```diff
@@ -68,7 +69,13 @@ class Identities {
   }
 
   static async createIdentity (options = {}) {
-    const keystore = options.keystore || Keystore.create(options.identityKeysPath || identityKeysPath)
+    if(!options.keystore) {
```
Comment (shamb0t) [inline src/identities.js]: Is it not possible to create a keystore here if none is passed?

Comment (aphelionz) [reply src/identities.js]: Fixed!

**Comment by shamb0t** [comment]
@aphelionz lgtm :+1:

**Pull request merged**